backend/Handlers.go

```diff
@@ -150,6 +155,30 @@ func Match(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(MatchList)
 }
 
+func RefreshCustomToken(w http.ResponseWriter, r *http.Request) {
+	params := mux.Vars(r)
+	accessToken := params["accessToken"]
+
+	lip, err := GetLiProfile(accessToken)
```

Comment:
why does he custom token refresh involve fetching the linkedin profile?

Reply:
I was doing it as a test if the user is authorized. Tom and I were speaking and maybe a better way of doing this would be to fetch the user's profile from Firebase, and then checking if the access token matches the one supplied by the user.